packages/esm-bed-management-app/src/ward-with-beds/ward-with-beds.component.tsx

```diff
@@ -114,6 +118,25 @@ const WardWithBeds: React.FC = () => {
               }>
               <span>Return to summary</span>
             </Button>
+            <span className={styles.backgroundDataFetchingIndicator}>
+              <span>{isValidating ? <InlineLoading /> : null}</span>
+            </span>
+            <Button
+              kind="ghost"
+              renderIcon={(props) => <Add size={16} {...props} />}
+              onClick={() => setShowAddBedModal(true)}>
+              {t('addBed', 'Add bed')}
```

Comment:
Wrapping this text in a `span` would get the padding similar to how it looks in the other button on hover:
```suggestion
             <span>{t('addBed', 'Add bed')}</span>
```
<img width="180" alt="Screenshot 2025-01-15 at 1 03 57 PM" src="https://github.com/user-attachments/assets/db3d2203-81e0-4599-8899-2e93fd068595" />
<img width="121" alt="Screenshot 2025-01-15 at 1 04 29 PM" src="https://github.com/user-attachments/assets/a1638017-0e8d-4029-80cb-3c7193fe9200" />